portal-impl/src/com/liferay/portal/minifier/MinifierUtil.java

```diff
@@ -111,6 +111,17 @@ private static String _minifyCss(String content) {
 
 			return unsyncStringWriter.toString();
 		}
+		catch (Throwable throwable) {
```

Comment:
Hey, why not replacing the catch for Exception above? :thinking:

Reply:
If this Error is thrown we will print all the content in the log under a [ERROR] trace. Per example: regarding the reported case by the customer it was like 20Mb of data, so maybe this could be dumping a lot of unnecessary information to the log.

If you prefer to replace the Exception by the throwable I'm ok with that, but I just have in mind this space consideration.

Reply:
Ah, OK, I see... The problem with that is that your algorithm is a bit flawed :sweat_smile: 

I mean, I ~like~ love the idea of not writing 20MB to the log, but what if a CSS of 20MB throws `Exception` because it's somehow malformed?

How about merging the two catches into one for `Throwable` then doing one of:

1. checking the length of the CSS content and not printing it if it exceeds some limit
2. printing only the N first bytes of the CSS content (where N is a reasonable number)
3. not printing the CSS content at all (in any case, the file should be available and I don't think it's so useful to see the CSS content in the log)

?

Reply:
Ok,
Maybe option 2, limiting to a megabyte? It gives enough context to know which css failed and doesn't flood the log

Reply:
Perfect :ok_hand: